docs/en/setup/backend/backend-aws-eks-monitoring.md

```diff
@@ -0,0 +1,265 @@
+| Network TX Error Count                | count/s | eks_cluster_node_pod_net_tx_error          | Instance   | Network RX error count of the pod running on the node        | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+| CPU Utilization                       | percent | eks_cluster_service_pod_cpu_utilization    | Endpoint   | The CPU Utilization of pod that belong to the service        | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+| Memory Utilization                    | percent | eks_cluster_service_pod_memory_utilization | Endpoint   | The Memory Utilization of pod that belong to the service     | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+| Network RX                            | bytes/s | eks_cluster_service_pod_net_rx_bytes       | Endpoint   | Network RX bytes of the pod that belong to the service       | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+| Network RX Error Count                | count/s | eks_cluster_service_pod_net_rx_error       | Endpoint   | Network TX error count of the pod that belongs to the service | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+| Network TX                            | bytes/s | eks_cluster_service_pod_net_tx_bytes       | Endpoint   | Network TX bytes of the pod that belong to the service       | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+| Network TX Error Count                | count/s | eks_cluster_node_pod_net_tx_error          | Endpoint   | Network TX error count of the pod that belongs to the service | [AWS Container Insights Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/awscontainerinsightreceiver/README.md) |
+
+### Customizations
+You can customize your own metrics/expression/dashboard panel.
+The metrics definition and expression rules are found in `/config/otel-rules/aws-eks/`.
+The AWS Cloud EKS dashboard panel configurations are found in `/config/ui-initialized-templates/aws_eks`.
+
+### AWS Container Insights Receiver Deploy Example
+
+```yaml
```

Comment:
This sample yaml is way too long that it doesn't highlight the important part, please remove unrelated content and replace with their official documentation link, an example would be 

```yaml
# ... other resources definitions

# OTEL configuration that specify OAP address("oap-service:11800") as otlp exporter address
apiVersion: v1
kind: ConfigMap
metadata:
  name: otel-agent-conf
  namespace: aws-otel-eks
  labels:
    app: opentelemetry
    component: otel-agent-conf
data:
  otel-agent-config: |
    # ...
    receivers:
      awscontainerinsightreceiver:

    # Make sure to add a job-name attribute
    processors:
      resource/job-name:
        attributes:
        - key: job_name   
          value: aws-cloud-eks-monitoring
          action: insert     

    exporters:
      otlp:
        endpoint: oap-service:11800
        tls:
          insecure: true
      logging:
          loglevel: debug          

    service:
      pipelines:
        metrics:
          receivers: [awscontainerinsightreceiver]
          processors: [resource/job-name]
          exporters: [otlp,logging]

# ... other resources

Reply:
I meant you should only highlight the important part here and link to the place where users can find the complete yaml, like https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awscontainerinsightreceiver#sample-configuration-for-container-insights

Reply:
If you want to give a complete sample, let's create a dedicated yaml file and link to that file, so users can simply use `kubectl apply -f sample.yaml` to set up the sample, without having to copy the large chunk of content.

Reply:
I think a complete OTEL YAML file would be good to host in the doc. Or we could link the file to the incoming file in the showcase.

Reply:
> I think a complete OTEL YAML file would be good to host in the doc

Agree with it, we don't have showcase for now.

Reply:
@pg-yang Are you going to move this to a separate file?

Reply:
I only provide a OTEL configuration sample in the doc